backend_new/src/services/email.service.ts

```diff
@@ -340,4 +329,75 @@ export class EmailService {
       }),
     );
   }
+
+  public async requestApproval(
+    jurisdictionId: IdDTO,
+    listingInfo: IdDTO,
+    emails: string[],
+    appUrl: string,
+  ) {
+    try {
+      const jurisdiction = await this.getJurisdiction([jurisdictionId]);
+      void (await this.loadTranslations(jurisdiction));
+      await this.send(
+        emails,
+        jurisdiction.emailFromAddress,
+        this.polyglot.t('requestApproval.header'),
+        this.template('request-approval')({
+          appOptions: { listingName: listingInfo.name },
+          appUrl: appUrl,
+          listingUrl: `${appUrl}/listings/${listingInfo.id}`,
+        }),
+      );
+    } catch (err) {
+      throw new HttpException('email failed', 500);
```

Comment:
do you think we should add a log here so we can know why this failed?

Reply:
I've added a log above this throw